saltstack/base/salt/top.sls

```diff
@@ -15,4 +15,5 @@ base:
     - fluent
 {% endif %}
     - ccm-client
+    - cis-controls
```

Comment:
```suggestion
{% if salt['environ.get']('INCLUDE_CIS') == 'Yes' %}
    - cis-controls
{% endif %}
```

This was lost on the move to the new PR. The idea here to be able to fully exclude the CIS benchmark stuff when building if necessary.

Reply:
@bormanbhaskar can you please address Kevin's comment and I can merge the PR?